src/ServicePulse.Host/vue/src/assets/main.css

```diff
@@ -2674,7 +2674,7 @@ span.fa-exclamation-triangle.danger {
     width: 100%;
     height: 90vh;
     margin-top: -100px;
-    background-image: url('../img/sp-loader.gif');
+    background-image: url('../a/img/sp-loader.gif');
```

Comment:
We should copy this image to the Vue assets folder

Reply:
There are a many such image files in the vue main css. I was planning to have all of them copied over late on. For now, I have copied the images of the ones mentioned in this PR .